src/Compilers/CSharp/Test/Emit3/Semantics/ExtensionTests2.cs

```diff
@@ -2285,5 +2285,71 @@ public static class E
             //         [System.Diagnostics.CodeAnalysis.UnscopedRef]
             Diagnostic(ErrorCode.ERR_UnscopedRefAttributeUnsupportedMemberTarget, "System.Diagnostics.CodeAnalysis.UnscopedRef").WithLocation(8, 10));
     }
+
+    [Fact]
+    public void Ambiguity_01()
+    {
+        var src = """
+var x = object.M; // 1
+x();
+
+System.Action y = object.M; // 2
+
+static class E1
+{
+    extension(object)
+    {
+        public static void M() { }
+    }
+}
+
+static class E2
+{
+    extension(object)
+    {
+        public static int M => 0;
+    }
+}
+""";
+
+        var comp = CreateCompilation(src);
+        comp.VerifyEmitDiagnostics(
+            // (1,9): error CS9286: 'object' does not contain a definition for 'M' and no accessible extension member 'M' for receiver of type 'object' could be found (are you missing a using directive or an assembly reference?)
```

Comment:
>// (1,9): error CS9286: 'object' does not contain a definition for 'M' and no accessible extension member 'M' for receiver of type 'object' could be found (are you missing a using directive or an assembly reference?) [](http://example.com/codeflow?start=12&length=216)

Are we tracking an improvement for diagnostics in this scenarios?

Reply:
Yes, we have a general tracking comment in `ResolveExtensions`: "// Tracked by [https://github.com/dotnet/roslyn/issues/76130](https://github.com/dotnet/roslyn/issues/76130) : we'll want to describe what went wrong in a useful way (see OverloadResolutionResult.ReportDiagnostics)"
I'll also add one for this specific test